lib/options.js

```diff
@@ -32,7 +32,9 @@ exports.parse = function parse(argv) {
     timeStampSupervisorLogs: true,
     syslog: false,
   };
-  var cluster = process.env.STRONGLOOP_CLUSTER;
+  // cluster_size is for compatibility with strong-cluster-control@1.x
+  var cluster = 'STRONGLOOP_CLUSTER' in process.env ?
```

Comment:
Yes, cluster_size was what strong-cluster-control used, via loadOptoins, via `rc`. Now, `slc run --cluster X` is equiv of `STRONGLOOP_CLUSTER=X`.